### ISSUE TITLE
[Bug]: Naming of Heading v. Header in Block Editor

### ISSUE THREAD

**Issue by badcat** [GetPublii/Publii#1622]
### Operating system

macOS Ventura 13.6.8

### Publii version

Version 0.46.0 (0.46.0)

### Post editor

Block editor

### Bug description

A bit pedantic, but in the current Block editor for both Posts and Pages, the tab inserter shows "Header". This should likely be "Heading" as header code is different than headings (h1, h2 etc).  Thanks for Publii!

### Steps to reproduce

* Click on Add new post (or page) dropdown, then "Use Block editor)
* Set a post/page title
* Press TAB key in content area, wait for dropdown list
<img width="339" alt="SCR-20240819-lxac" src="https://github.com/user-attachments/assets/37daeedc-f00e-4dd6-bb0b-7f9a5337b023">

**Comment by bobmitro** [comment]
You're right! It should be "Heading".  This change will be implemented in the upcoming Publii 0.46.4 release. 
Thanks for your feedback!
